contracts/PushCore/PushCoreV3.sol

```diff
@@ -296,27 +295,38 @@ contract PushCoreV3 is
         // Check channel's current state
         bytes32 _channelBytesID = BaseHelper.addressToBytes32(msg.sender);
 
+           if(channelInfo[_channelBytesID].channelState == 2) {
+                if (_amount < ADD_CHANNEL_MIN_FEES) {
+                 revert Errors.InvalidArg_LessThanExpected(ADD_CHANNEL_MIN_FEES, _amount);
+                }
+
+                IERC20(PUSH_TOKEN_ADDRESS).safeTransferFrom(msg.sender, address(this), _amount);
+                reactivateChanel( _channelBytesID, _amount);
+           }else {
+              deactivateDeleteChannel( _channelBytesID, msg.sender);
+            }     
+    }
+
+    function deactivateDeleteChannel(bytes32 _channelBytesID, address recipient) internal {
```

Comment:
deactivateDeleteChannel name doesn't sound very intuitive
my suggestion would be to change it to deactiveChannel and it'd be self-explanatory that it can be used to delete timebound channel as well